contracts/sol/QuadVoteTallyVerifierMedium.sol

```diff
@@ -165,21 +165,21 @@ contract QuadVoteTallyVerifierMedium {
+        vk.delta2 = Pairing.G2Point([uint256(19781296222358353286912814199294057381520495233655268581758073491044543931788),uint256(15441813332966757358665007622767656435873993749013870427858798699836248112040)], [uint256(10479770894907623494834804044098186663061940425846417308266742461618406411074),uint256(15686235373657679450584915904107324567313998249766873490575746670840280216853)]);
+        vk.IC[0] = Pairing.G1Point(uint256(2977545462639543101608782132446798528679035159024046056105166121243737504055),uint256(10976552791013863974760508642652783847591166611809288370096549374809259670328));
+        vk.IC[1] = Pairing.G1Point(uint256(16617117652719723525567739882546231852126896289564440288656781634351934373036),uint256(7858043334357141243829884899107161964407668676997805099432989477137316782154));
+        vk.IC[2] = Pairing.G1Point(uint256(21034780521515524634732714444946311911257864754357440769235423173183712930103),uint256(14989237583681163119607720897491464874702187894252534220522903041923666715136));
+        vk.IC[3] = Pairing.G1Point(uint256(10067594776052824527165810506011231130912779482663133864912723057370489845492),uint256(10820075793848169251020567601314068841841218933066087065630771339447183744397));
+        vk.IC[4] = Pairing.G1Point(uint256(12318998056847047854413314288994458073238001476534821128047529278896400078509),uint256(21045525366690692356049933185363977937079390356416540417018799012434932908671));
+        vk.IC[5] = Pairing.G1Point(uint256(1006362880820666291343426807174131485654423662363691446979907404834533996937),uint256(10229387861108374514909559612177121576910000345854012267697085502571775622454));
+        vk.IC[6] = Pairing.G1Point(uint256(1265862285389912583354988517188240046545245531899484378775932237585919113405),uint256(19533887002306214906401588301741821164134268055785360059716974096750249574592));
+        vk.IC[7] = Pairing.G1Point(uint256(9366383303037262111495176025900763714531644320956547136527308721507028908668),uint256(12342056557140844024941014535132352840593653924946843771623130612151334486161));
+        vk.IC[8] = Pairing.G1Point(uint256(377648027856046402067856432085235248427633732903195773131155890402237030240),uint256(13859189692549416893530466171559207483847454708965810295259420276885186194111));
+        vk.IC[9] = Pairing.G1Point(uint256(12137642909261162531614282794577276988027441374171514839983505649086044434171),uint256(18386481830849357503396854902661055139744772247970949541726767736810048014090));
+        vk.IC[10] = Pairing.G1Point(uint256(11168589902532857753647687088832653482107030818840713069138157464421493318222),uint256(14379320319015633530046109708947298315994836438043323861191851455013758947415));
```

Comment:
I think verifiers can be removed & gitignored because the maci-circuits package does not contain corresponding snark parameters.

Reply:
Absolutely! I'll remove them and  update .gitignore.